tests/test_filter_datasets.py

```diff
@@ -294,3 +341,109 @@ def test_stratified_crossval_splits_multiple_attributes(many_stimuli, crossval_f
     assert sum(len(f.x) for f in train_fixations) == (crossval_folds - val_folds - test_folds) * len(fixations.x)
 
     assert len(train_stimuli) == crossval_folds
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva', 1, False)])
+def test_filter_stimuli_by_attribute_dva(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    fixations = fixation_trains[:]
+    filtered_stimuli, filtered_fixations = filter_stimuli_by_attribute(file_stimuli_with_attributes, fixations, attribute_name, attribute_value, invert_match)
+    if attribute_name == 'dva' and attribute_value == 1 and invert_match is False:
```

Comment:
why do you still test those values? The condition will always be true, right? You can just remove the if-clause and always run the next lines

Reply:
yes will remove them